docs/user/check-ref/kubernetes_wrapper.rst

```diff
@@ -56,8 +56,8 @@ Methods of Kubernetes
         :param ready: Pod readiness status. If ``None`` then all pods are returned.
         :type ready: bool
 
-        :param **kwargs: Pod :ref:`labelSelectors` filters.
-        :type **kwargs: dict
+        :param \**kwargs: Pod :ref:`labelSelectors` filters.
```

Comment:
I think it would make more sense if we changed to ``kwargs`` instead. That way we keep it consistent with normal RST format.